docs/contributing/contribution/development.md

```diff
@@ -0,0 +1,179 @@
+# Community Developer Guide
+
+```mermaid
+flowchart TD
+  check((Chat, Ask,<br>Investigate)) --> fork[Fork<br>the Project]
+  fork --> |code| code[Implement]
+  code --> |comment| done{Working?}
+  done --> |code| code
```

Comment:
We already have a `code` step between `fork` and `implement`. Should this be `no` given that the other option is `yes`?

Reply:
You are right, it's confusing. let me push a review of this

Reply:
Check 4316cde